applications/mobu/templates/deployment.yaml

```diff
@@ -82,23 +111,33 @@ spec:
                 - ALL
             readOnlyRootFilesystem: true
           volumeMounts:
-            {{- if .Values.config.autostart }}
-            - name: "autostart"
+            - name: "config"
               mountPath: "/etc/mobu"
               readOnly: true
-            {{- end }}
             - name: "tmp"
               mountPath: "/tmp"
       securityContext:
         runAsNonRoot: true
         runAsUser: 1000
         runAsGroup: 1000
       volumes:
-        {{- if .Values.config.autostart }}
-        - name: "autostart"
-          configMap:
-            name: {{ include "mobu.fullname" . }}-autostart
-        {{- end }}
+        - name: "config"
```

Comment:
Am I reading this right, that we're collapsing the two configmaps into one?

The only reason we might not want to do this is that on-the-fly editing of the autostart configmap is reasonably common (to modify or disable flocks) so this might be a little more dangerous.

Reply:
Nope, this is [mounting the two separate ConfigMaps as two separate files](https://kubernetes.io/docs/concepts/storage/projected-volumes/) in to the same `/etc/mobu` directory:

```
appuser@mobu-6f947f55cc-jrrfk:/$ ls -l /etc/mobu/
total 0
lrwxrwxrwx 1 root root 21 Jun 25 15:39 autostart.yaml -> ..data/autostart.yaml
lrwxrwxrwx 1 root root 25 Jun 25 15:39 github-config.yaml -> ..data/github-config.yaml
appuser@mobu-6f947f55cc-jrrfk:/$
```
The autostart CM can still be manually edited independently, and the changes do make it into the running mobu pods (though we'd have to create new mobu pods to pick up the changes anyway, right?)